app/views/host_mailer/host_built.html.erb

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1;">
+<body>
+<style type="text/css">
+  a, a:visited {
+    color: #6bb5df !important;
+  }
+
+  a:hover {
+    color: #CCCCCC !important;
+  }
+</style>
+<div style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1;">
+  <table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="bodyTable" bgcolor="#f1f1f1">
+    <tr>
+      <td align="center" valign="top">
+        <table border="0" id="emailContainer" style="max-width: 600px;">
+          <tr>
+            <td align="center" valign="top">
+              <div align="left">
+                <h3 style="font-weight: normal;"><%= _("<b>FOREMAN</b> BUILD COMPLETE").html_safe %></h3>
```

Comment:
Styling is also slightly different to the audits summary, where this title is 120% font size.  I haven't compared it against all the others..